src/main/kotlin/uk/gov/dluhc/printapi/service/ElectorDocumentRemovalDateResolver.kt

```diff
@@ -92,7 +92,7 @@ class ElectorDocumentRemovalDateResolver(
     }
 
     private fun getTotalDaysForWorkingDays(issueDate: LocalDate, requiredWorkingDays: Int, gssCode: String): Int {
-        val upcomingBankHolidays = bankHolidayDataClient.getBankHolidayDates(BankHolidayDivision.fromGssCode(gssCode))
+        val upcomingBankHolidays = bankHolidayDataClient.getBankHolidayDates(BankHolidayDivision.fromGssCode(gssCode), issueDate)
```

Comment:
There's already a second argument that defaults to today (which is the problem).

Yes I'm going to look at VCA too.

Reply:
In that case this is a thing of beauty! 🤣

Reply:
Just to confirm, a fix isn't required for VCA because the `removalDateTime` is set when the application is determined there and then (i.e. "now", rather than being related to another date - such as issueDate).